kanidmd/src/lib/idm/server.rs

```diff
@@ -67,6 +70,7 @@ pub struct IdmServer {
     async_tx: Sender<DelayedAction>,
     // Our webauthn verifier/config
     webauthn: Webauthn<WebauthnDomainConfig>,
+    pw_badlist_cache: Arc<CowCell<BTreeSet<Value>>>,
```

Comment:
This should be HashSet<String> I think since during any password operation we have the password cleartext as a string first, and that's more compact in the HashSet.